medcat/ner/transformers_ner.py

```diff
@@ -76,9 +76,8 @@ def __init__(self, cdb, config: Optional[ConfigTransformersNER] = None,
         else:
             self.training_arguments = training_arguments
 
-
     def create_eval_pipeline(self):
-        self.ner_pipe = pipeline(model=self.model, task="ner", tokenizer=self.tokenizer.hf_tokenizer)
+        self.ner_pipe = pipeline(model=self.model, task="ner", tokenizer=self.tokenizer.hf_tokenizer,stride=self.config.general['chunking_overlap_window'])
```

Comment:
to confirm - without the stride kwarg it does no stride at all, and with 0 it strides but with no overlap?

Reply:
Exactly, without stride, no chunking - it would not de_id after 512
With stride 0, chunking with no overlap